forge-gui/res/cardsfolder/upcoming/caught_in_the_crossfire.txt

```diff
@@ -0,0 +1,7 @@
+Name:Caught in the Crossfire
+ManaCost:R R
+Types:Instant
+A:SP$ Charm | Choices$ DBDamageOutlaw,DBDamageNonOutlaw | MinCharmNum$ 1 | CharmNum$ 2 | Spree$ True
+SVar:DBDamageOutlaw:DB$ DamageAll | SpreeCost$ 1 | NumDmg$ 2 | ValidCards$ Creature.Outlaw | SpellDescription$ CARDNAME deals 2 damage to each outlaw creature. (Assassins, Mercenaries, Pirates, Rogues, and Warlocks are outlaws.)
+SVar:DBDamageNonOutlaw:DB$ DamageAll | SpreeCost$ 1 | NumDmg$ 2 | ValidCards$ Creature.nonOutlaw | SpellDescription$ CARDNAME deals 2 damage to each non-outlaw creature.
```

Comment:
wrong valid since it's not real type

Reply:
Should it be Creature.!Outlaw

Reply:
Tested it and `Creature.!Outlaw` works. Also you need `K:Spree` at the top (under the type line) so the additional costs for the choices are actually added.